README.md

```diff
@@ -151,11 +158,16 @@ More information about the extension concept can be found here [TBW].
 
 More information about shadowJar can be found [here](https://github.com/johnrengelman/shadow).
 
-## Generate the OpenApi specification
+### Generate the OpenApi specification
 
-Please refer to [this document](./openapi.md).
+Please refer to [this document](openapi.md).
 
-# Directory structure
+### Code style & Patterns
```

Comment:
This seems also something to be put in the developer documentation

Reply:
You mean the `styleguide.md` in general?

Reply:
Agree on removing this section from the `README` as this is part of the `CONTRIBUTING.md`

Reply:
Added some sentences to the `developer/README.md`